feat(COCO-STQA-68)Added Selection to Hall of Fame

CarbonFanatic opened the pull request:
Added a hardcoded Select in the Hall of fame.
Does not work yet

CarbonFanatic committed:
Added Selection to Hall of Fame

Added a hardcoded Select in the Hall of fame.
Does not work yet

CarbonFanatic committed:
PetController, getPetTypes

GetMapping Creates new method that returns a collection of all pet types to the url ending with /pets/petTyps

CarbonFanatic committed:
Revert "Added Selection to Hall of Fame"

This reverts commit 2a489212103917e23af332c6df71e7b82dd0c912.

CarbonFanatic committed:
Auto Populate Dropdown

Select Statement gets auto populated with all existing PetTypes.
no longer Hard Coded

CarbonFanatic committed:
Added some Raiting to pets

Added some Rating to pets.
Began the Js code for sorting pets

CarbonFanatic committed:
Have filtering workig.

Need to remove some entries if only 1 or 2 pets are found.
Need to have All selection working.

Need to sort the hall of fame by rating/timesrated.

Shayminn commented on src/main/webapp/WEB-INF/jsp/welcome.jsp:
```diff
@@ -23,14 +22,26 @@
-                <th class="text-center">1st</th>
-                <th class="text-center">2nd</th>
-                <th class="text-center">3rd</th>
+                <th id="HOFRank1" class="text-center">1st</th>
```
Are these IDs necessary for something else?

Shayminn commented on src/main/webapp/WEB-INF/jsp/welcome.jsp:
```diff
@@ -23,14 +22,26 @@
             <tr id="ranking">
-                <th class="text-center">1st</th>
-                <th class="text-center">2nd</th>
-                <th class="text-center">3rd</th>
```
Are the following IDs necessary for something else?

Shayminn reviewed (commented):
Is the functionality working as of right now?
Do you need help with anything? Is anything stopping you from completing this story, if it is still not completed?

CarbonFanatic committed:
Finished filtering of pets

Added a test case in PetControllerTest for getPetTypes

Still need to chan variable name so they make sense.
Possible sellenium test case

CarbonFanatic committed:
Added comments fix var names

Added Comments and fixed variable names

CarbonFanatic committed:
Added PetTypeSelect test case

Create a test case for the select drop down

Shayminn committed:
Merge branch 'master' into feat/COCO-STQA-68_Set_Filters

NichitaMantaluta reviewed (approved):
Everything looks good to me

CarbonFanatic commented:
Was forced to Merge with master multiple time so the history is a little all over the place

CarbonFanatic committed:
Merge branch 'master' into feat/COCO-STQA-68_Set_Filters

Shayminn reviewed (approved):
Code reviewed, added some comments, everything else looks good!
